manheim_c7n_tools/policygen.py

```diff
@@ -100,6 +85,82 @@ def run(self):
         logger.info('Writing region list to regions.rst...')
         self._write_file('regions.rst', self._regions_rst())
 
+    def _load_defaults(self):
+        """
+        Load a defaults.yml file from either the ``policies/`` subdirectory
+        or directories in the ``policy_source_paths`` configuration key.
+        """
+        defaults = None
+        # read the global defaults
+        if os.path.exists(os.path.join('policies', 'defaults.yml')):
+            defaults = self._read_file_yaml(
+                os.path.join('policies', 'defaults.yml')
+            )
+
+        # check policy folders for defaults
+        try:
+            paths = self._config.policy_source_paths
```

Comment:
I imagine this was left over from a refactor... `paths` is only used once here, so this line and the next could be condensed down into `for path in self._config.policy_source_paths:`